### PR TITLE
Add more verbose information in build-apks command.

### PR DESCRIPTION
**Pull request by aliscafo**
https://github.com/google/bundletool/issues/169

### CODE REVIEW

File: src/main/java/com/android/tools/build/bundletool/io/ApkSerializerManager.java [hunk 1]
```diff
@@ -191,7 +198,12 @@ public void populateApkSetBuilder(
                         (ModuleSplit split) -> {
                           ZipPath apkPath = apkPathManager.getApkPath(split);
                           return executorService.submit(
-                              () -> apkSerializer.serialize(split, apkPath));
+                              () -> {
+                                ImmutableList<ApkDescription> list =
+                                    apkSerializer.serialize(split, apkPath);
+                                logVerboseInfo(split, apkPath);
```
Comment (plecesne) [inline src/main/java/com/android/tools/build/bundletool/io/ApkSerializerManager.java:204]: Sorry, I realized that I suggested this location, but actually at the end of the  .serialize() method is probably better.

File: src/main/java/com/android/tools/build/bundletool/commands/BuildApksCommand.java [hunk 2]
```diff
@@ -731,6 +742,15 @@ public static CommandHelp help() {
...
                     InstallApksCommand.COMMAND_NAME)
                 .build())
+        .addFlag(
+            FlagDescription.builder()
+                .setFlagName(VERBOSE_FLAG.getName())
+                .setOptional(true)
+                .setDescription(
+                    "If enabled, verbose information about what is happening "
+                        + "during the command execution will be displayed.",
+                    InstallApksCommand.COMMAND_NAME)
```
Comment (plecesne) [inline src/main/java/com/android/tools/build/bundletool/commands/BuildApksCommand.java:752]: Also, remove this.

File: src/main/java/com/android/tools/build/bundletool/io/ApkSerializerManager.java [hunk 3]
```diff
@@ -80,17 +83,21 @@
...
       ListeningExecutorService executorService,
       ApkListener apkListener,
       ApkModifier apkModifier,
+      boolean verbose,
       int firstVariantNumber) {
     this.appBundle = appBundle;
     this.apkSetBuilder = apkSetBuilder;
     this.apkPathManager = new ApkPathManager();
     this.executorService = executorService;
+    this.verbose = verbose;
```
Comment (plecesne) [inline src/main/java/com/android/tools/build/bundletool/io/ApkSerializerManager.java:100]: nit: Move this line two below to keep the same order as the constructor parameters (and ideally the same order as how the fields are defined in the class -- although I realize they're already not in order).

**Comment by aliscafo** [comment]
@googlebot I signed it!

File: src/main/java/com/android/tools/build/bundletool/io/ApkSerializerManager.java [hunk 4]
```diff
@@ -265,6 +282,17 @@ public void populateApkSetBuilder(
         .collect(toImmutableList());
   }
 
+  private void logVerboseInfo(ModuleSplit split, ZipPath apkPath) {
+    DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
+    if (verbose) {
+      System.out.printf(
+          "INFO: [%s] '%s' of type '%s' was written to the disk.%n",
+          LocalDateTime.now().format(formatter),
```
Comment (plecesne) [inline src/main/java/com/android/tools/build/bundletool/io/ApkSerializerManager.java:290]: Use: LocalDateTime.now(ZoneId.systemDefault()) 
to be explicit about which timezone is used.
We have a check that enforces it internally: https://errorprone.info/bugpattern/JavaTimeDefaultTimeZone

**Pull request merged**